1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,168 @@
+Feature: User edits Project
+    In order to document a project 
+    As a logged in User
+    I want to edit existing and create new projects
+
+    @2903a
+    Scenario Outline: User sees the Project Wizard
+        Given User opens My Projects
+        When they click Create New Project Button
+        Then the User is the Project Owner
+        And they see tabs:
+            | Description  |
+            | Participants |
+            | Financing    |
+        And they see the Description tab
+        And they see fields:
+            | Title                    |
+            | Summary                  |
+            | Coordinating Institution |
+            | Start Date               |
+            | End Date                 |
+            | Internal Project Code    |
+        And they see a Support Button
+        And they see a Button "<Button>" decided by Project's "<Status>"
+        # Project  API demands a PM to allow a POST, use of this button
+        # will allways give a error msg: "Go to next page and add a PM"
```

Comment:
Det var kanskje vel pessimistisk da :p
Det vil vel bare skje om man skal opprette nytt prosjekt, og trykker "Lagre kladd" i stedet for "Neste" på første fane?
Hvis man skal endre prosjekt vet vi at Prosjektet allerede har en PM, så man kan fint lagre allerede her.
Hvis man skal opprette nytt prosjekt vil jeg tippe at de fleste trykker "Neste" i stedet for "Lagre kladd". Hvis man da legger til PM, og navigerer tilbake hit vil også lagring gå fint (om jeg har skjønt det riktig)

```suggestion
```

Reply:
Ja, der finnes tilfeller der det vil fungere - eg vart ledet av "When they click Create New Project Button" i samme scanario.